objectivec/README.md

```diff
@@ -200,23 +200,30 @@ supported keys are:
         entry can be made as "no_package:PATH=prefix", where PATH is the
       path for the .proto file.
 
-  * `use_package_as_prefix` and `proto_package_prefix_exceptions_path`: The
-    `value` for `use_package_as_prefix` can be `yes` or `no`, and indicates
-    if a prefix should be derived from the proto package for all the symbols
-    for files that don't have the `objc_class_prefix` file option (mentioned
-    above). This helps ensure the symbols are more unique and means there is
-    less chance of ObjC class name collisions.
+  * `use_package_as_prefix`, `package_as_prefix_forced_prefix` and
+    `proto_package_prefix_exceptions_path`: The `value` for
+    `use_package_as_prefix` can be `yes` or `no`, and indicates if a prefix
+    should be derived from the proto package for all the symbols for files that
+    don't have the `objc_class_prefix` file option (mentioned above). This helps
+    ensure the symbols are more unique and means there is less chance of ObjC
+    class name collisions.
 
     To help in migrating code to using this support,
     `proto_package_prefix_exceptions_path` can be used to provide the path
     to a file that contains proto package names (one per line, comments allowed
     if prefixed with `#`). These package won't get the derived prefix, allowing
     migrations to the behavior one proto package at a time across a code base.
 
+    `package_as_prefix_forced_prefix` can be used to provide a value that will
```

Comment:
I found this name a little confusing at first. Would it make sense to just call it something like `forced_package_prefix`?

Reply:
Since it only apples if package_as_prefix is also enable, that's why i left it there.  my expectation is it won't be that common to use, so I'm good with the slightly more verbose name to hopefully better indicate how it is used.

I can also look at adding a validation that this isn't set when the option isn't set; but it still feels like erroring on the side of the longer name is better.